spec/controllers/inbox_controller_spec.rb

```diff
@@ -8,13 +8,15 @@
   describe "GET #show" do
     it "redirects to user page when not logged in" do
       get :show, user_id: user.login
-      it_redirects_to user_path(user)
```

Comment:
Could these be rewritten with parentheses to be consistent with the majority of the other examples? Also, the descriptions should be updated to mention the flash messages.